php-nginx/php.ini

```diff
@@ -478,7 +478,7 @@ error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT
 ; Development Value: On
 ; Production Value: Off
 ; http://php.net/display-errors
-display_errors = Off
+display_errors = stderr
```

Comment:
Hmm, I see your point. Unfortunately, this is the only way (I know of) to
separate stdout and stderr in Php. That warning is probably meant for when
display_errors is set to On.

Also, I'd be fine with setting log_errors to On and combining them. So it
depends on how important we think it is to separate them. Personally I'd
like to try this out and see if it works as expected.
On Thu, Jan 7, 2016 at 8:23 AM Takashi Matsuo <notifications@github.com>
wrote:

> In php-nginx/php.ini
> <https://github.com/GoogleCloudPlatform/php-docker/pull/41#discussion_r49091445>
> :
>
> > @@ -478,7 +478,7 @@ error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT
> >  ; Development Value: On
> >  ; Production Value: Off
> >  ; http://php.net/display-errors
> > -display_errors = Off
> > +display_errors = stderr
>
> FYI Travis build is failing because phpinfo.php shows a warning like:
>
> Warning: phpinfo() has been disabled for security reasons in /app/phpinfo.php on line 19
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/php-docker/pull/41/files#r49091445>
> .
>